paasta_tools/iptables.py

```diff
@@ -0,0 +1,113 @@
+"""iptables helper functions.
+
+Unlike the `firewall` module, these functions know nothing about PaaSTA and
+could effectively be a third-party library. They just make working with
+iptables a little bit easier.
```

Comment:
Great if it works. Yeah I was mostly thinking about something that'd be able to use a code interface rather than the iptables executable, so great you found one.